atom/common/api/atom_bindings.cc

```diff
@@ -125,6 +126,35 @@ void AtomBindings::Hang() {
     base::PlatformThread::Sleep(base::TimeDelta::FromSeconds(1));
 }
 
+// static
+v8::Local<v8::Value> AtomBindings::GetHeapStatistics(v8::Isolate* isolate) {
+  v8::HeapStatistics v8_heap_stats = {};
```

Comment:
should not be required to do "= {}". Default constructor of HeapStatistics is doing the same.

Reply:
good point, changed that